config/BUILD.bazel

```diff
@@ -55,3 +57,18 @@ genrule(
     tools = ["@io_k8s_sigs_kustomize//:kustomize"],
     visibility = ["//visibility:public"],
 )
+
+genrule(
+    name = "credential_file",
+    outs = ["credentials"],
+    visibility = ["//visibility:public"],
+    cmd = " && ".join([
+        "touch $@",
+        "export AWS_ACCESS_KEY_ID=$$(grep ^AWS_ACCESS_KEY_ID bazel-out/volatile-status.txt | cut -f2 -d\" \")",
+        "export AWS_SECRET_ACCESS_KEY=$$(grep ^AWS_SECRET_ACCESS_KEY bazel-out/volatile-status.txt | cut -f2 -d\" \")",
```

Comment:
Will this cause issues with local dev workflows by assuming creds are always in bazel-out/volatile-status.txt?

Or am I potentially misreading what this is doing?

Reply:
volatile status gets repopulated from env when you run go test, so basically it's just assuming these are in the user's environment. I don't really know how else to get the creds inside, though it seems like @randomvariable is changing how this works anyway?

Reply:
May be able to use bazel configs as switches to choose where it looks. Too much work right now I think though.

Reply:
Re-reading, will need to support AWS_SESSION_TOKEN too, but can fix up in follow up PR.